### PR TITLE
Harness now reports a difference in line endings when comparing baselines

### PR DESCRIPTION
**Pull request by JsonFreeman**

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
:ship: :it:

File: tests/baselines/reference/newLineFlagWithCRLF.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-//// [newLineFlagWithCRLF.ts]
+//// [newLineFlagWithCRLF.ts]
 var x=1;
```
Comment (mhegazy) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: this file should be line ending sensitive.

File: src/harness/harness.ts [hunk 2]
```diff
@@ -1702,9 +1702,9 @@ module Harness {
                 expected = IO.readFile(refFileName);
             }
 
-            var lineEndingSensitive = opts && opts.LineEndingSensitive;
+            var lineEndingInsensitive = opts && opts.LineEndingSensitive === false; // default is true
```
Comment (mhegazy) [inline src/harness/harness.ts]: Who sets this flag? i could not find it used anywhere... if that is correct.. then remove this whole section instead.

File: tests/baselines/reference/newLineFlagWithCRLF.js (hunk 1 above)
Comment (DanielRosenwasser) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: It is, isn't it?

File: src/harness/harness.ts (hunk 2 above)
Comment (JsonFreeman) [inline src/harness/harness.ts]: I did not find it either. You're right, I should remove it.

File: tests/baselines/reference/newLineFlagWithCRLF.js (hunk 1 above)
Comment (JsonFreeman) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: @mhegazy I'm not sure what you mean.

Comment (mhegazy) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: sorry missed the change in name of the variable.. can you also make sure that these files are never normalized.. or if you are deleting the code, then we are fine :)

Comment (JsonFreeman) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: I plan to delete the code.

Comment (JsonFreeman) [inline tests/baselines/reference/newLineFlagWithCRLF.js]: I've now deleted the code.

**Comment by mhegazy** [comment]
:+1:

**Comment by yuit** [comment]
:+1:

**Pull request merged**